docs/development.md

```diff
@@ -82,6 +82,8 @@ ocamlmerlin -version  # just checking ocamlmerlin is in your PATH
 code .
 ```
 
+For Python, Pycharm is good.
```

Comment:
Nit: not really sure how to use this information unless we include more Pycharm setup docs.

Reply:
it's just a regular Python project so there's really no setup. I just don't want people to try to use the objectively worse VSCode python extension since we say "VSCode" above.